KSM tests are (randomly) failing on the ARM Ci

```
Entering KSM settle mode on first container
No need to wait for KSM to settle
Pages merged 0
Entering KSM settle mode on second container
No need to wait for KSM to settle
Pages merged 0
Comparing merged pages between containers
ERROR: The merged pages on the second container is less than the first container
time="2021-04-20T18:59:50Z" level=warning msg="task test1 exit with non-zero exit code 9"
time="2021-04-20T18:59:50Z" level=warning msg="task test exit with non-zero exit code 9"
restoring KSM settings
Makefile:127: recipe for target 'ksm' failed
make: *** [ksm] Error 1
Failed at 35: sudo -E PATH="$PATH" CRI_RUNTIME="containerd" bash -c "make ksm"
Build step 'Execute shell' marked build as failure
```

I'm not sure what's going on there, see one example of a job failure: http://jenkins.katacontainers.io/job/kata-containers-2.0-ubuntu-ARM-PR/1244/

/cc @GabyCT @c3d

--- a/integration/ksm/ksm_test.sh
+++ b/integration/ksm/ksm_test.sh
@@ -19,6 +19,12 @@ KATA_KSM_THROTTLER="${KATA_KSM_THROTTLER:-yes}"
 PAYLOAD_ARGS="${PAYLOAD_ARGS:-tail -f /dev/null}"
 IMAGE="${IMAGE:-quay.io/prometheus/busybox:latest}"
 WAIT_TIME="60"
+arch=$("${dir_path}"/../../.ci/kata-arch.sh -d)
+
+if [ "$arch" == "aarch64" ]; then
+	echo "Skip KSM test: $arch does not support it"
+	exit 0
+fi
 
 function setup() {
 	sudo systemctl restart containerd

